hss_exercise.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2020 D-Wave Systems Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## ------- import packages -------
+import networkx as nx
+import dimod
+# TODO:  Add code here to import your sampler
+
+# TODO:  Add code here to import your Traveling Salesman QUBO generator
+
+
+## ------- provided distance matrix -------
+distance_matrix = [
+        [0, 2230, 1631, 1566, 1346, 1352, 1204],
+        [2230, 0, 845, 707, 1001, 947, 1484],
+        [1631, 845, 0, 627, 773, 424, 644],
+        [1566, 707, 627, 0, 302, 341, 1027],
+        [1346, 1001, 773, 302, 0, 368, 916],
+        [1352, 947, 424, 341, 368, 0, 702],
+        [1204, 1484, 644, 1027, 916, 702, 0],
+    ]
+
+
+def get_token():
```

Comment:
Why does this exercise have a token and the other one doesn't?

Reply:
@arcondello, the token is for using the LeapHybridSampler. In the other exercise, I want them to use a non-token solver, like SimulatedAnnealing or TabuSampler. In this one, they should use LeapHybridSampler, and it needs the token.

Reply:
That's not true. LeapHybridSampler does not need a token if they have a generic leap account

Reply:
Hi, I think this one's OK - the token is needed.